native/commonizer/src/org/jetbrains/kotlin/descriptors/commonizer/konan/NativeDistributionCommonizer.kt

```diff
@@ -127,20 +127,20 @@ class NativeDistributionCommonizer(
         val parameters = CommonizerParameters(statsCollector, ::logProgress).apply {
             val storageManager = LockBasedStorageManager("Commonized modules")
 
-            resultsConsumer = NativeDistributionResultsConsumer(
-                repository = repository,
-                originalLibraries = allLibraries,
-                destination = destination,
-                copyStdlib = copyStdlib,
-                copyEndorsedLibs = copyEndorsedLibs,
-                logProgress = ::logProgress
-            )
-            dependeeModulesProvider = NativeDistributionModulesProvider.forStandardLibrary(storageManager, allLibraries.stdlib)
+        resultsConsumer = NativeDistributionResultsConsumer(
```

Comment:
Is the indent correct here?

Reply:
Sorry. Rebase is happened to be broken.. This file is not supposed to be changed at all.